libs/react-helpers/src/lib/get-events.ts

```diff
@@ -0,0 +1,21 @@
+import type { Schema } from '@vegaprotocol/types';
+
+type Event = {
+  __typename?: string;
+};
+
+type BusEvent<T> = {
```

Comment:
Question (non-blocking): Wouldn't it be better to constrain `T` here already as `<T extends Event>`?

Reply:
```suggestion
type BusEvent<T extends Event> = {
```